pkg/driver/controller.go

```diff
@@ -162,8 +167,28 @@ func (d *Driver) CreateVolume(ctx context.Context, req *csi.CreateVolumeRequest)
 		basePath = value
 	}
 
+	if value, ok := volumeParams[AzName]; ok {
+		azName = value
+	}
+
+	// Fetch aws role ARN for cross account mount from secrets
```

Comment:
I feel like this part needs more commend, how/where the cross-account mount credential is retrieved and what is secrets?

Reply:
I added information on how to setup the secrets in the description of the PR.

You'd need to pass the secret into storage class parameter and [external-provisioner](https://github.com/kubernetes-csi/external-provisioner) will pull the information and adds it to `CreateVolume` request.

Reply:
I plan to add some documentation on readMe on how it works.